app/src/main/java/com/google/android/samples/socialite/widget/ui/ZeroState.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.samples.socialite.widget.ui
+
+import android.content.Context
+import androidx.compose.runtime.Composable
+import androidx.glance.GlanceModifier
+import androidx.glance.appwidget.lazy.LazyColumn
+import androidx.glance.layout.Alignment
+import androidx.glance.layout.Box
+import androidx.glance.layout.fillMaxSize
+import com.google.android.samples.socialite.model.Contact
+import com.google.android.samples.socialite.widget.model.WidgetModel
+import com.google.android.samples.socialite.widget.model.WidgetModelRepository
+import kotlinx.coroutines.runBlocking
+
+@Composable
+fun ZeroState(repository: WidgetModelRepository, widgetId: Int, context: Context) {
```

Comment:
Can you get the context from `LocalContext` rather than having to pass it here as an argument?

This should also accept a modifier.